.travis/molecule.sh

```diff
@@ -1,12 +1,11 @@
 #!/usr/bin/env bash
 
-# deploy/cluster_role_binding.yaml specify the namespace: default
-# The default namespace on molecule is: osdk-test
-# For running the molecule test we should:
-# - Option 1: change the namespace at deploy/cluster_role_binding.yaml to: osdk-test
-# - Option 2: change the namespace at molecule test to: default
-# As molecule provides TEST_OPERATOR_NAMESPACE we will use the 2nd option:
-export TEST_OPERATOR_NAMESPACE=default
+curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.8.1/kind-linux-amd64
+chmod +x ./kind
+sudo mv ./kind /usr/local/bin/kind
+
+curl -s "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"  | bash
+sudo mv ./kustomize /usr/local/bin/kustomize
 
 echo "Starting molecule test"
-molecule test -s test-local
+molecule test -s kind
```

Comment:
So this replaces the old 3 different types of tests?

Reply:
if you mean molecule tests, yes!